### DISCUSSION TITLE
Your request has expired, please try again

### DISCUSSION THREAD

**Discussion by Lr-2002** [community/community#63354] [GitHub Education] [answered]
### Hi everyone,

I'm applying for my edu pack on Andriod using VPN (I'm in another city rather than my school's location )
And When I pushed the button "take a picture" 
The web showed as below:
"Your application cannot be reviewed until you fix the following:"
 Your request has expired, Please try again.

I've retried for a few times and the info showed again.

Any Way to solve it?

**Comment by arjunkumarbose** [comment, marked as answer]
- Just wait for a while, don't use any VPN in such verification cases, which can reject your application as well if anything goes wrong.
- If your WIFI doesn't work then try to do so using Mobile Internet, as ISP sometimes doesn’t fetch properly.
- Here is another discussion that might help: https://github.com/orgs/community/discussions/37159%20)

**Comment by Lr-2002** [comment]
For me , the problem was solved seemed because the Chrome has no authority to read the camera
The seem process was ok in my system web viewer

**Reply by arjunkumarbose** [reply]
Great as you've fixed it! :D
